src/js/gi/components/Breadcrumbs.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+class Breadcrumbs extends React.Component {
+
+  renderDefault(props) {
+    return (
+      <li className="active">{props.currentLabel}</li>
+    );
+  }
+
+  renderNamed(props) {
+    return (
+      <span>
+        <li><a href="/gi-bill-comparison-tool/">GI Bill Comparison Tool</a></li>
```

Comment:
This should probably be a `Link` from `react-router`. Any of this app's routes should use `Link` rather than `a`, or else the links will load the full page.

Also, to avoid prefixing all the internal links with `/gi-bill-comparison-tool`, you can do something like this: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/rx/rx-entry.jsx#L16